src/planner/binder/statement/bind_show.cpp

```diff
@@ -0,0 +1,26 @@
+#include "duckdb/planner/binder.hpp"
+#include "duckdb/parser/statement/show_statement.hpp"
+#include "duckdb/planner/operator/logical_show.hpp"
+
+using namespace duckdb;
+using namespace std;
+
+BoundStatement Binder::Bind(ShowStatement &stmt) {
+
+	BoundStatement result;
+
+  auto plan = Bind(*stmt.info->query);
+  stmt.info->types = plan.types;
+  stmt.info->aliases = plan.names;
+	
+	auto logical_plan_unopt = plan.plan->ToString();
```

Comment:
I think this variable is unnecessary, no?

Reply:
Correct. I removed it.